src/api/search/search.ts

```diff
@@ -0,0 +1,17 @@
+import axios from '..';
```

Comment:
moved this here from seeds/search
we need to move more constructs such as search criteria in here but after removing 'seeds' from the names... that will be a follow up